app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -57,6 +57,60 @@ examples:
         text: "Use GOV.UK Verify"
         hint_text: "You'll have an account if you've already proved your identity with a certified company, such as the Post Office."
         bold: true
+  with_hint_on_form_group:
+    data:
+      name: "radio-group-error"
+      id_prefix: "hint"
+      hint: "You’ll need to prove your identity using one of the following methods"
+      items:
+      - value: "government-gateway"
+        text: "Use Government Gateway"
+      - value: "govuk-verify"
+        text: "Use GOV.UK Verify"
+  with_heading:
+    description: This adds a legend element containing the text supplied.
+    data:
+      name: "radio-group-heading"
+      heading: "Are you hungry?"
+      items:
+      - value: "yes"
+        text: "Yes"
+      - value: "no"
+        text: "No"
+  with_heading_and_hint:
+    data:
+      name: "radio-group-heading"
+      heading: "What is your favourite colour?"
+      hint: "If your favourite is not below, pick the colour closest to it."
+      items:
+      - value: "red"
+        text: "Red"
+      - value: "green"
+        text: "Green"
+      - value: "blue"
+        text: "Blue"
+  with_page_heading:
+    description: This adds a H1 element containing the text supplied.
+    data:
+      name: "radio-group-heading"
+      heading: "Is it raining?"
+      is_page_heading: true
+      items:
+      - value: "yes"
+        text: "Yes"
+      - value: "no"
+        text: "No"
+  with_page_heading_and_hint:
+    data:
+      name: "radio-group-heading"
+      heading: "Is it snowing?"
+      is_page_heading: true
+      hint: "Sleet or hail doesn’t count."
+      items:
+      - value: "yes"
+        text: "Yes"
+      - value: "no"
+        text: "No"
   with_hint_text:
```

Comment:
Would be worth renaming this as something like `with_hint_text_on_radios` to match the form of `with_hint_on_form_group`